browser/main/lib/ConfigManager.js

```diff
@@ -122,7 +122,7 @@ function set (updates) {
     : 'default'
 
   if (newTheme !== 'default') {
-    editorTheme.setAttribute('href', '../node_modules/codemirror/theme/' + newTheme + '.css')
+    editorTheme.setAttribute('href', '../node_modules/codemirror/theme/' + newTheme.split(' ')[0] + '.css')
```

Comment:
This fixing is just only for solarized theme.
Can you add checking that `newTheme` is solarized or not?

Reply:
Yes, this change is for solarized theme only. Currently, solarized dark/light is the only the theme which contains space, so this change will not cause the problem for other theme.

However, as you pointed out, if I add the `newTheme` is solarized or not, this makes more clear the intention of code.

Following code also doesn't check the theme, so I will also update this.
https://github.com/BoostIO/Boostnote/blob/993cb9cb0b7c72b13b1a1cba5ed26c7a29fbcdb7/browser/components/MarkdownPreview.js#L261

Thanks for reviewing!